### PR TITLE
First Proposal of the Learning Session Manager API

### PR DESCRIPTION
**Pull request by gulyx**

### CODE REVIEW

File: lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/lsm/rest/ManageLeanringSessions.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.QueryParam;
+
+import eu.learnpad.exception.LpRestException;
+
+public interface ManageLeanringSessions {
```
Comment (woshilapin) [inline lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/lsm/rest/ManageLeanringSessions.java]: You mean `ManageLearningSessions`? ;-)

**Comment by gulyx** [comment]
Right sorry. Can you deal with that?

**Comment by woshilapin** [comment]
OK, I'll merge that right now.

**Comment by woshilapin** [comment]
Pull request merged and I've changed the name of the class.

**Pull request merged**